src/main.c

```diff
@@ -40,6 +42,8 @@ main(int argc, char **argv, char **envp)
     if (argc < 2)
 	error(EX_USAGE, 0, "missing command to record");
 
+    record_start("output.txt");
```

Comment:
Use `build-recorder.out` as it's less likely to conflict with anything produced by the actual commands being run.